examples/lib/defaultHtmlStepUi.js

```diff
@@ -310,11 +310,23 @@ function DefaultHtmlStepUi(_sequencer, options) {
 
     $stepAll('.download-btn').on('click', () => {
 
+      function dataURLtoBlob(dataurl) {
+        let arr = dataurl.split(','), mime = arr[0].match(/:(.*?);/)[1],
+          bstr = atob(arr[1]), n = bstr.length, u8arr = new Uint8Array(n);
+        while(n--){
+          u8arr[n] = bstr.charCodeAt(n);
+        }
+        return new Blob([u8arr], {type:mime});
+      }
+      
       var element = document.createElement('a');
-      element.setAttribute('href', step.output);
+      
       element.setAttribute('download', step.name + '.' + fileExtension(step.imgElement.src));
       element.style.display = 'none';
       document.body.appendChild(element);
+      var blob = dataURLtoBlob(step.output);
+      var objurl = URL.createObjectURL(blob);
```

Comment:
What is the difference between this URL and dataURL?

Reply:
I just encountered this in https://github.com/jywarren/bookletize.js and data urls have a length limit in browsers, so they fail on large files, and also blob is more efficient because they don't need to be reencoded. 

I think this is probably worth it!

Reply:
Can we internally use blobs too, then?

Reply:
I think it's possible. We had very very long ago considered different formats for between-module data handoff, including possibly buffers...? i forget, but it was in our very first issue: https://github.com/publiclab/image-sequencer/pull/1#issuecomment-288275324

We could open a new issue with this idea -- basically that default is data-urls, because strings are completely compatible, but if two modules can negotiate that both are compatible with blobs, that's OK too. I think the data-urls worked especially well because they had to be displayed anyways. But now that I think of it I think it's possible to display a blob as well in the browser. Please feel free to open an issue to discuss this!

Reply:
Also note https://stackoverflow.com/questions/7650587/using-javascript-to-display-a-blob

Reply:
I wonder if this would just work, but it depends on the run environment, I imagine:

https://github.com/publiclab/image-sequencer/blob/5298d1e0ecadb83a02111342a72dc3690b27a6ad/src/Run.js#L18-L54

We could add a parameter to drawStep somewhere that evaluates whether the neighboring steps can both handle blobs.

Reply:
I broke this into an issue at https://github.com/publiclab/image-sequencer/issues/1847 !!